test/warp.jl

```diff
@@ -387,4 +387,40 @@ ref_img_pyramid_grid = Float64[
             @test nearlysame(round.(Float64.(imgrq_cntr[axes(imgrq_cntr)...]), digits=3), round.(ref_img_pyramid_quad, digits=3))
         end
     end
+
+    @testset "imrotate" begin
+        # imrotate is a wrapper, hence only need to guarantee the interface works as expected
+
+        test_types = (Float32, Float64, N0f8, N0f16)
+        graybar = repeat(range(0,stop=1,length=100),1,100)
+        @testset "interface" begin
+            for T in test_types
+                img = Gray{T}.(graybar)
+                @test_nowarn imrotate(img, 45)
```

Comment:
why `45` if its in radians? What are you testing here?

Reply:
It's a mistake here, in the first version of `imrotate` I didn't use radians but degrees. I'll change this.